cluster/etcd-stack.yaml

```diff
@@ -189,7 +189,12 @@ Resources:
       IpProtocol: tcp
       FromPort: 2379
       ToPort: 2479
+{{- if eq .Cluster.ConfigItems.etcd_stack_security_group_temp "false"}}
       SourceSecurityGroupId: !GetAtt EtcdSecurityGroup.GroupId
+{{- end }}
+{{- if eq .Cluster.ConfigItems.etcd_stack_security_group_temp "true"}}
+      SourceSecurityGroupId: !GetAtt EtcdSecurityGroupTemp.GroupId
+{{- end }}
```

Comment:
Maybe it's better to copy the whole `EtcdIngressMembers` resource into `EtcdIngressMembersTemp` so you can have both Security Groups with all settings at the same time, otherwise I expect you break one security group during the switch.

Reply:
But can we have two `AWS::EC2::SecurityGroupIngress` resources?

Reply:
I simplified it.